Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -76,11 +69,17 @@ public static string OAuthToken
     [Export(typeof(IGitPlugin))]
     public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin
     {
+        public static string GitHubAuthorizationRelativeUrl = "authorizations";
         public static string UpstreamConventionName = "upstream";
+        public readonly StringSetting GitHubApiEndpoint = new StringSetting("GitHub (Enterprise) api endpoint", "https://api.github.com");
```

Comment:
```suggestion
        public readonly StringSetting GitHubApiEndpoint = new StringSetting("GitHub (Enterprise) API endpoint", "https://api.github.com");
```

Reply:
I didn't had the time to test the last changes I made. I pushed it in case I eventually found some time over the weekendtto test it on my personal computer. 

So, please wait a little more before merging it.